### PR TITLE
live: Call webbrowser.open

### PR DESCRIPTION
**Pull request by daavoo**
* [X] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [X] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

Closes iterative/dvclive#180

### CODE REVIEW

File: dvc/repo/live.py [hunk 1]
```diff
@@ -10,15 +14,27 @@
     from dvc.path_info import PathInfo
     from dvc.repo import Repo
 
+webbrowser_open = call_only_once(webbrowser.open)
```
Comment (skshetry) [inline dvc/repo/live.py:17]: You can use [`funcy.once`](https://funcy.readthedocs.io/en/stable/flow.html#once) instead. Btw, is it possible to have multiple `live` outputs? How does it work with this?

Comment (daavoo) [reply dvc/repo/live.py:17]: Didn't know about funcy.once, thanks!

>  Btw, is it possible to have multiple live outputs?

It is not possible. A stage can only have 1 `live` output. There might be multiple plots in the `live` output directory but that doesn't affect the number of times the HTML generation is called.

Comment (skshetry) [reply dvc/repo/live.py]: But can multiple stages have multiple live outputs?

Comment (daavoo) [reply dvc/repo/live.py]: Yes. 

I assume that you are pointing to better use `once_per_args`?

File: dvc/repo/live.py [hunk 2]
```diff
@@ -10,15 +15,27 @@
...
+    index_path = render(
+        out.repo, plots, metrics=metrics, path=html_path, refresh_seconds=5
+    )
+
+    url = index_path.as_uri()
+
+    if "Microsoft" in uname().release:
+        url = Path(index_path) / "index.html"
+
+    webbrowser_open(url)
```
Comment (skshetry) [inline dvc/repo/live.py:38]: Could you please move this piece to `ui`? Maybe as `ui.launch` (similar to [`click.launch`](https://click.palletsprojects.com/en/8.0.x/utils/#launching-applications)) or `ui.open_browser`? We can handle all `relpath`, `uri` and `WSL` stuff inside it. What do you think?

Comment (daavoo) [reply dvc/repo/live.py:38]: Sounds good

Comment (daavoo) [reply dvc/repo/live.py:38]: Opened #6993

**Review by karajan1001 — APPROVED** [review]
LGTM

File: tests/func/test_live.py [hunk 3]
```diff
@@ -237,6 +237,7 @@ def test_live_checkpoints_resume(
 
 def test_dvc_generates_html_during_run(tmp_dir, dvc, mocker, live_stage):
     show_spy = mocker.spy(dvc.live, "show")
+    webbrowser_open = mocker.patch("dvc.repo.live.webbrowser_open")
```
Comment (skshetry) [inline tests/func/test_live.py:240]: @daavoo, it seems that the above change affected all of the tests and now opens the browser while running the tests.

**Pull request merged**